src/interfaces/store-interfaces/state-model-interface.ts

```diff
@@ -1,11 +1,12 @@
-import { AccountInformationIF, AddressesIF, BusinessInformationIF, CertifyIF,
-  CorrectionInformationIF, DocumentDeliveryIF, EffectiveDateTimeIF, EntitySnapshotIF,
-  FeesIF, FilingDataIF, PeopleAndRolesIF, NameRequestIF, NameTranslationIF,
-  NewAlterationIF, RestorationStateIF, ShareStructureIF, StateFilingRestorationIF, TombStoneIF,
-  ValidationFlagsIF,
-  RulesMemorandumIF } from '@/interfaces/'
-import { CompletingPartyIF, ContactPointIF, SpecialResolutionIF, StaffPaymentIF }
-  from '@bcrs-shared-components/interfaces/'
+import {
+  AccountInformationIF, AddressesIF, BusinessInformationIF, CertifyIF, CorrectionInformationIF,
+  DocumentDeliveryIF, EffectiveDateTimeIF, EntitySnapshotIF, FeesIF, FilingDataIF, PeopleAndRolesIF,
```

Comment:
Super small thing (i know it was previously unsorted), but we can move the PeopleAndRolesIF forward and bring the RulesMemorandumIF backward to get these sorted 😆

Reply:
You bet. Done.

Reply:
Thanks! Resolving.